src/main/java/com/ProjectEureka/backend/models/Question.java

```diff
@@ -0,0 +1,53 @@
+package com.ProjectEureka.backend.models;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Document(collection = "Questions")
+public class Question {
+    @Id
+    private String id;
+
+    private String title;
+
+    @Field("question_date")
+    private String questionDate;
```

Comment:
Ideally, we want this to be of type `Date` with also a `@CreatedDate` from `java.util.Date` to automatically add the date for us.  Please also apply it to the answers model.

Reply:
done in the latest commit